software/webhook-tester.yml

```
@@ -0,0 +1,12 @@
+name: "WebHook Tester"
+source_code_url: "https://github.com/tarampampam/webhook-tester"
+website_url: "https://github.com/tarampampam/webhook-tester"
+description: "Powerful tool for testing WebHooks and more."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Go
+  - deb
```

```suggestion
  - deb
  - k8s
```

https://github.com/tarampampam/webhook-tester/pull/630